todo.md

```
@@ -0,0 +1,99 @@
+# TODO List for HDR Image Creator
+
+## 🚀 **Project Setup**
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)